src/components/balloon.js

```diff
@@ -1,61 +1,89 @@
-                        <img src={fireSmall} alt="" className="balloon__fire-img balloon__fire-img--small"/>
+            <div className={classnames('balloon', {
+                'balloon--animation-reset': this.props.animationState === 'ANIMATION_RESET',
+                'balloon--animation-in': this.props.animationState === 'ANIMATION_START',
+                'balloon--animation-out': this.props.animationState === 'ANIMATION_GO_TO',
+            })}>
+
+                <div className={classnames('balloon__wrapper', {
+                    'balloon__wrapper--animation-out': this.props.animationState === 'ANIMATION_GO_TO'
+                })} onAnimationEnd={this.closeContainerAnimation}>
+
+                    <div className={classnames('balloon__wrapper-pulse', {
+                        'balloon__wrapper-pulse--animation-out': this.props.animationState === 'ANIMATION_GO_TO',
+                    })}>
+
+                        <img src={balloonImg} alt=""
+                             className={classnames('balloon__img balloon__img--balloon', {
+                                 'balloon__img--animation-reset': this.props.animationState === 'ANIMATION_RESET',
+                                 'balloon__img--animation-in': this.props.animationState === 'ANIMATION_START',
+                                 'balloon__img--animation-out': this.props.animationState === 'ANIMATION_GO_TO',
+                             })}/>
+
+                        <div className="balloon__fire">
+                            <Flame src={fireBig} size='big' animationState={this.props.animationState}/>
+                            <Flame src={fireMedium} size='medium' animationState={this.props.animationState}/>
+                            <Flame src={fireSmall} size='small' animationState={this.props.animationState}/>
+                        </div>
                     </div>
                 </div>
 
-                <img src={cloud} alt="" className="balloon__img balloon__img--cloud balloon__img--cloud-left"/>
-                <img src={cloud} alt="" className="balloon__img balloon__img--cloud balloon__img--cloud-right"/>
-                <img src={cloud} alt="" className="balloon__img balloon__img--cloud balloon__img--cloud-behind"/>
+                <div className={classnames('balloon__clouds', {
+                    'balloon__clouds--animation-reset': this.props.animationState === 'ANIMATION_RESET',
+                    'balloon__clouds--animation-in': this.props.animationState === 'ANIMATION_START',
+                    'balloon__clouds--animation-out': this.props.animationState === 'ANIMATION_GO_TO',
+                })}>
+                    <CloudsContainer
+                        containerSelector='.main-content'
+                        animationState={this.props.animationState}
+                        render={(props) => (
+                            <Fragment>
+                                <Cloud position='right' multiplier='4.5'
```

Comment:
You're passing string value as a multiplier. Add PropTypes in CloudComponent to prevent this kind of errors.